common/src/main/java/earth/terrarium/cadmus/common/commands/claims/ClaimSettingsCommand.java

```diff
@@ -0,0 +1,263 @@
+package earth.terrarium.cadmus.common.commands.claims;
+
+import com.mojang.brigadier.CommandDispatcher;
+import com.mojang.brigadier.arguments.BoolArgumentType;
+import com.mojang.brigadier.builder.ArgumentBuilder;
+import com.mojang.brigadier.builder.LiteralArgumentBuilder;
+import com.teamresourceful.resourcefullib.common.utils.CommonUtils;
+import com.teamresourceful.resourcefullib.common.utils.TriState;
+import com.teamresourceful.resourcefullib.common.utils.modinfo.ModInfoUtils;
+import earth.terrarium.cadmus.api.teams.TeamProviderApi;
+import earth.terrarium.cadmus.common.claims.CadmusDataHandler;
+import earth.terrarium.cadmus.common.claims.ClaimSettings;
+import earth.terrarium.cadmus.common.compat.prometheus.CadmusAutoCompletes;
+import earth.terrarium.cadmus.common.compat.prometheus.PrometheusIntegration;
+import earth.terrarium.cadmus.common.teams.TeamHelper;
+import earth.terrarium.cadmus.common.util.ModGameRules;
+import earth.terrarium.cadmus.common.util.ModUtils;
+import net.minecraft.commands.CommandSourceStack;
+import net.minecraft.commands.Commands;
+import net.minecraft.network.chat.Component;
+import net.minecraft.server.level.ServerPlayer;
+import net.minecraft.world.level.GameRules;
+
+public class ClaimSettingsCommand {
+
+    public static void register(CommandDispatcher<CommandSourceStack> dispatcher) {
+        dispatcher.register(Commands.literal("claim")
+            .then(Commands.literal("settings")
+                .then(canBreak())
+                .then(canPlace())
+                .then(canExplode())
+                .then(canInteractWithBlocks())
+                .then(canInteractWithEntities())
+                .then(canDamageEntities())
+            ));
+    }
+
+    private static ArgumentBuilder<CommandSourceStack, LiteralArgumentBuilder<CommandSourceStack>> canBreak() {
+        return Commands.literal("canBreak")
+            .then(Commands.argument("value", BoolArgumentType.bool())
+                .executes(context -> {
+                    ServerPlayer player = context.getSource().getPlayerOrException();
+                    CommandHelper.runAction(() -> {
+                        if (!checkPrometheusPermissions(player, CadmusAutoCompletes.BLOCK_BREAKING, ModGameRules.RULE_DO_CLAIMED_BLOCK_BREAKING)) {
+                            throw ClaimException.NOT_ALLOWED_TO_MANAGE_SETTINGS;
+                        }
+
+                        String id = TeamHelper.getTeamId(player.getServer(), player.getUUID());
+
+                        if (ModUtils.isTeam(id) && !TeamProviderApi.API.getSelected().canModifySettings(id, player)) {
+                            throw ClaimException.NOT_ALLOWED_TO_MANAGE_SETTINGS;
+                        }
+
+                        boolean canBreak = BoolArgumentType.getBool(context, "value");
```

Comment:
the argument should be a tristate to allow the player to set it back to the default, DO NOTE: do not let the admins set theirs to a non boolean.